portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/* Shows about text when the "About" button is pressed. */
+function showAbout () {
+    const aboutContainer = document.getElementById('aboutSpace');
+    aboutTest = aboutContainer.getElementsByTagName('p');
+    if (aboutTest.length > 0) {
+        aboutText = aboutContainer.getElementsByTagName('p')[0];
+    }
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    // Change the text in the aboutSpace element 
+    if (aboutText.className === "about-hide") {
+        aboutText.className = "about-show";
+        aboutContainer.className = "background";
+    }
+    
+    else {
+        //aboutContainer.innerText = "";
+        aboutText.className = "about-hide";
+        aboutContainer.className = "";
+    }
+}
+
+/* Shows pictures when the "Pictures" button is pressed. */
+function showPics() {
+    const imgs = ["ucla1.JPG", "ucla2.jpg.JPG", "book.JPG", "harrypotter.JPG"]
+    const srcBase = "https://8080-0ba309ed-2f3c-4b55-83b6-7741ff94bec3.us-west1.cloudshell.dev/base"
```

Comment:
Rather than having a special `base` value, I think it would be cleaner to either set the `src` to "" (empty string) to represent no current image, or just make the decision to hide/show based off the class names that currently exist and assign a new `src` value when showing, but leave `src` as-is when hiding, since it doesn't matter when the image is hidden.

Reply:
Good idea, this made it cleaner. Thanks!